packages/console/src/components/Uploader.tsx

```diff
@@ -91,10 +92,11 @@ export const Done = ({ dataCID }: DoneProps): JSX.Element => {
   const [, { setFile }] = useUploader()
   const cid: string = dataCID?.toString() ?? ''
   return (
-    <div className='flex flex-col items-center w-full'>
+    <div className='flex flex-col items-center w-full '>
```

Comment:
request: don't add this space

```suggestion
    <div className='flex flex-col items-center w-full'>
```